4-running-scripts/README.md

```diff
@@ -66,14 +66,19 @@ Test-Connection -ComputerName "example.com" -Count 3 -Delay 2 -TTL 255 -BufferSi
 Next we run the script using `bolt script run`. This will upload the script to all specified nodes, ensure it's executable and finally run it, returning the output to the console.
 
 ```
-$ bolt script run testconnection.ps1 --nodes winrm://<node> --user <user> --password <password>
-node_1:
-
-Source        Destination     IPV4Address      IPV6Address                              Bytes    Time(ms)
-------        -----------     -----------      -----------                              -----    --------
-DESKTOP-G9... example.com     93.184.216.34                                             256      83
-DESKTOP-G9... example.com     93.184.216.34                                             256      85
-DESKTOP-G9... example.com     93.184.216.34                                             256      86
+$ bolt script run testconnection.ps1 --n winrm://vagrant:vagrant@localhost:55985
+Started on localhost...
```

Comment:
This output should reference `vagrant`, yeah?

Reply:
the host is localhost